api/src/utils/get-config-from-env.ts

```diff
@@ -1,29 +1,37 @@
 import { useEnv } from '@directus/env';
+import { toArray } from '@directus/utils';
 import camelcase from 'camelcase';
 import { set } from 'lodash-es';
 
-export function getConfigFromEnv(
-	prefix: string,
-	omitPrefix?: string | string[],
-	type: 'camelcase' | 'underscore' = 'camelcase',
-): Record<string, any> {
+export interface GetConfigFromEnvOptions {
+	omitPrefix?: string | string[];
+	omitKey?: string | string[];
+	type?: 'camelcase' | 'underscore';
+}
+
+export function getConfigFromEnv(prefix: string, options?: GetConfigFromEnvOptions): Record<string, any> {
 	const env = useEnv();
+	const type = options?.type ?? 'camelcase';
 
 	const config: any = {};
 
+	const lowerCasePrefix = prefix.toLowerCase();
```

Comment:
Thoughts on optimizing by performing the `toLowerCase` for the `options.omitPrefix ` and `options.omitKey` here?